java-checks/src/main/java/org/sonar/java/checks/tests/AssertionArgumentOrderCheck.java

```diff
@@ -59,12 +65,17 @@
   protected MethodMatchers getMethodInvocationMatchers() {
     return MethodMatchers.or(
       MethodMatchers.create().ofTypes(ORG_JUNIT_ASSERT)
```

Comment:
I am not sure whether these matchers could be used in other places. If so, might be a nice candidate for moving to Utils/Helpers

Reply:
This matcher is related to 4 specific methods of `org.junit.Assert` that have an expected and an actual argument. I didn't find this exact need in other rules related to `org.junit.Assert`, so I keep this matcher here for now.